### ISSUE TITLE
Unrecognized data stream while reading image file

### ISSUE THREAD

**Issue by HoxtaH** [bmaltais/kohya_ss#1718]
Been trying to train a simple 768x768 SD 1.5 Lora in kohya recently and have been running into the error below.  In the past I was able to train Lora for SD 1.5 and SDXL fairly easy.  Im not sure what the problem is now.  Has anyone one else had this issue recently after updates?  Ive seen some that look close to my issue in other post but not quite.  It basically makes it all the way to the end of the "trainer script" and then errors out when its passed the actual argument.  The "caching latents" bar below can change depending on when it crashes.  

I have already tried uninstalling kohya and reinstalling.  I have also tried to roll it back to a specific "sha" commit that worked for me in the past but the UI acts buggy now when you downgrade versions.   Any assistance would be appreciated

![image](https://github.com/bmaltais/kohya_ss/assets/18668780/b4820d0a-0641-4d79-96e2-0cf325a08c09)

**Comment by UrbanLegendXV** [comment]
Did you ever get a solve for this?

**Comment by HoxtaH** [comment]
> Did you ever get a solve for this?

Yes actually.  I forgot to close this issue.  For me it was actually an odd thing with data.  Some of the files I used said they were png but they actually werent.  Kohya's training algorithm when it comes to caching latents can recognize the difference between what says its a png but reads as a jpg.  This leads to the error above.  When i changed out my dataset completely the error resolved itself.  

lol the amount of research to find this error was insane but I hope this helps you.
